Cargo.toml

```diff
@@ -21,11 +21,11 @@ maintenance = { status = "passively-maintained" }
 repository = "cardoe/cargo-bitbake"
 
 [dependencies]
-cargo = "^0.41"
-git2 = "0.10"
+cargo = "^0.49"
+git2 = "0.13"
 itertools = "^0.5.0"
 lazy_static = "^1"
 md5 = "^0.3"
 regex = "^1"
 structopt = "^0.3"
-failure = "0.1"
+anyhow = "^1.0"
```

Comment:
this should go to the top because (except for the old `failure`) this list is sorted alphabetically